governance/votes/Executive vote - November 29, 2023.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Template - [Executive Vote] PSM-GUSD-A Removal from Cash Stablecoins, Coinbase Custody Debt Ceiling Increase, Various Parameter Changes, RWA Foundation Dao Resolution, Andromeda Legal Expenses, Vest Streams Updates and Spark Proxy-Spell - November 29, 2023
```

Comment:
See our comment below re: cash stablecoins.

Suggested changed wording for vesting portion of title

```suggestion
title: Template - [Executive Vote] PSM-GUSD-A Debt Ceiling Reduction, Coinbase Custody Debt Ceiling Increase, Various Parameter Changes, RWA Foundation Dao Resolution, Andromeda Legal Expenses, Vesting Stream Updates and Spark Proxy-Spell - November 29, 2023
```